src/get-started/editor.md

```diff
@@ -41,92 +50,132 @@ that's OK, skip ahead to the [next step: Test drive][].
 
 ## Install VS Code
 
-VS Code is a lightweight editor with complete Flutter
-app execution and debug support.
+[VS Code][] is a code editor to build and debug apps.
+With the Flutter plugin installed, you can compile, deploy, and debug
+Flutter apps.
+
+To install the latest version of VS Code,
+follow Microsoft's instructions for the relevant platform:
+
+- [Install on macOS][]
+- [Install on Windows][]
+- [Install on Linux][]
+
+[VS Code]: https://code.visualstudio.com/
+[Install on macOS]: https://code.visualstudio.com/docs/setup/mac
+[Install on Windows]: https://code.visualstudio.com/docs/setup/windows
+[Install on Linux]: https://code.visualstudio.com/docs/setup/linux
+
+## Install the VS Code Flutter plugin
+
+1. Start **VS Code**.
+
+1. Go to **View** <span aria-label="and then">></span>
+   **Command Palette...**.
+
+   You can also press <kbd>Control</kbd> / <kbd>Command</kbd> +
+   <kbd>Shift</kbd> + <kbd>P</kbd>.
+
+1. Type `install`.
+
+1. Select **Extensions: Install Extensions**.
+
+1. Type `flutter` in the extensions search field.
+
+1. Select **Flutter** in the list.
+
+1. Click **Install**.
+   Installing the Flutter plugin also installs the Dart plugin.
 
-* [VS Code][], latest stable version
+## Validate your VS Code setup with the Flutter Doctor
```

Comment:
The instructions will discuss how to validate. It's usually not necessary for someone clicking on a header to know the how yet.

```suggestion
## Validate your VS Code setup
```